flight/PiOS/Common/pios_bmp280.c

```diff
@@ -0,0 +1,441 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_BMP280 BMP280 Functions
+ * @brief Hardware functions to deal with the altitude pressure sensor
+ * @{
+ *
+ * @file       pios_bmp280.c
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2012-2014
```

Comment:
But was it based on some code that was copyrighted by Tau? That's the relevant factor. :wink: